examples/follow_path.jl

```diff
@@ -2,12 +2,12 @@ using Javis, Animations
 
 function ground(args...)
     background("white")
-    sethue("black")
+    return sethue("black")
```

Comment:
Why did you add `return` here?

Reply:
I prefer `sethue("black")` without the `return` as it's not needed afterwars.

Reply:
it was added by the formatter

Reply:
if you don't want to set returns at the bottom of all functions, is should be specified in the Formatter TOML I believe

Reply:
also, see:
https://github.com/invenia/BlueStyle#method-definitions
it is advised by the blue style, that's why it's there by default